setup.cfg

```diff
@@ -0,0 +1,3 @@
+[pep8]
+ignore = E11,E101,E121
```

Comment:
Why these errors ?
E11: fix indentation. This should be here I suppose
E101: good, since we have max-line-length (btw, I just did a pep8 config file myself, and it works with `max_line_length`, did you try `max-line-length` ?)
E121: can be fixed with `indent_size=2`

Here is the one I came up with:
```
[pep8]
# Pep8 codes:
# E111 - indentation is not a multiple of four
# E114  indentation is not a multiple of four (comment)
#
# autopep8 codes:
# E101 - reindent all lines: removed because max_line_length
# E501 - Try to make lines fit within --max-line-length characters: removed because max_line_length
ignore=E111,E114
max_line_length=120
# indent_size is unknown to pep8, but not to autopep8
indent_size=2
```
I also have a file that is I think a good template. If the editor of a developers changes it, then it means he has bad formating options (I'll do a PR soon)

Reply:
https://github.com/DIRACGrid/DIRAC/pull/3519

Reply:
I will remove this file, as your PR is more precise.